cmd/podman/inspect.go

```diff
@@ -77,6 +78,9 @@ func inspectCmd(c *cli.Context) error {
 		inspectType = inspectTypeContainer
 	}
 	outputFormat := c.String("format")
+	if strings.ToUpper(outputFormat) == formats.IDString {
```

Comment:
I believe we should do 
```
if strings.Contains(outputFormat, "{{.Id}}") {
	outputFormat = strings.Replace(outputFormat, "{{.Id}}", formats.IDString, -1)
}
```
because it is possible for the user to do `--format "{{.Id}} {{.ContainerConfig}}"`

Reply:
SGTM